core/src/main/java/org/elasticsearch/cluster/routing/allocation/RebalanceDecision.java

```diff
@@ -111,7 +123,7 @@ public String getExplanation() {
 
     @Override
     public void innerToXContent(XContentBuilder builder, Params params) throws IOException {
-        super.toXContent(builder, params);
+        builder.field("current_node_ranking", currentNodeRanking);
```

Comment:
I wonder if we should just call this `weight_ranking` and put it under the toXContent for the currently assigned node.